src/commands/prepare.ts

```diff
@@ -5,7 +5,6 @@ import simpleGit, { SimpleGit } from 'simple-git';
 import { Arguments, Argv, CommandBuilder } from 'yargs';
 
 import {
-  checkMinimalConfigVersion,
```

Comment:
You sure you want to do this? Performing this check only under the new `config` command significantly weakens the protection against version mismatch.

Reply:
I'm not moving this under the new command, I'm moving it to the core `getConfiguration()` method which needs o be called by _every_ command that does anything useful. So we are increasing he protection :)